tests/test_repository.py

```diff
@@ -51,6 +51,8 @@
             "jeos-firstboot",
             # has a boolean dependency on the kernel
             "ecryptfs-utils",
+            # has a boolean dependency on xwayland
+            "at-spi2-core-2",
```

Comment:
The package name is different:
```suggestion
            # has a boolean dependency on xwayland
            "at-spi2-core",
```